README.md

```diff
@@ -1,17 +1,18 @@
-# amazon-q-connectjs
```

Comment:
meant to remove the space here?

Reply:
I don't think so, probably want it to be an anchor link. Will fix it!